packages/combo-box/src/vaadin-combo-box-data-provider-mixin.js

```diff
@@ -81,20 +70,35 @@ export const ComboBoxDataProviderMixin = (superClass) =>
       ];
     }
 
+    constructor() {
+      super();
+
+      /** @type {DataProviderController} */
+      this._dataProviderController = new DataProviderController(this, {
```

Comment:
If `_dataProviderController` is a private property, it should be prefixed with a double underscore.
```suggestion
      this.__dataProviderController = new DataProviderController(this, {
```

Reply:
Done.